src/style.css

```
@@ -1,3 +1,79 @@
+/* stylelint-disable font-family-name-quotes */
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+}
+
+.logo {
+  text-decoration: none;
+  height: 40px;
+  padding: 10px 0;
+
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  height: 100%;
+  margin-left: 20px;
+}
+
+.nav__item:first-child{
+  margin-left: 0;
+}
+
+.nav__item a {
+  display: flex;
+  align-items: center;
+  height: 100%;
+  text-decoration: none;
+  color: #000;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Roboto";
```

remove the disable line and add the generic-font
```suggestion
  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
  font-family: "Roboto", sans-serif;
```